aptos-move/block-executor/src/view.rs

```diff
@@ -391,33 +538,75 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> TResourceGr
     type Layout = MoveTypeLayout;
     type ResourceTag = T::Tag;
 
-    fn resource_group_size(&self, _group_key: &Self::GroupKey) -> anyhow::Result<u64> {
-        unimplemented!("TResourceGroupView not yet implemented");
+    fn resource_group_size(&self, group_key: &Self::GroupKey) -> anyhow::Result<u64> {
+        let mut group_read = match &self.latest_view {
+            ViewState::Sync(state) => state.read_group_size(group_key, self.txn_idx)?,
+            ViewState::Unsync(state) => state.unsync_map.get_group_size(group_key)?,
+        };
+
+        if matches!(group_read, GroupReadResult::Uninitialized) {
+            self.initialize_group(group_key)?;
+
+            group_read = match &self.latest_view {
+                ViewState::Sync(state) => state.read_group_size(group_key, self.txn_idx)?,
+                ViewState::Unsync(state) => state.unsync_map.get_group_size(group_key)?,
+            }
+        };
+
+        Ok(group_read.into_size())
     }
 
     fn get_resource_from_group(
         &self,
-        _group_key: &Self::GroupKey,
-        _resource_tag: &Self::ResourceTag,
+        group_key: &Self::GroupKey,
+        resource_tag: &Self::ResourceTag,
         _maybe_layout: Option<&Self::Layout>,
     ) -> anyhow::Result<Option<Bytes>> {
-        unimplemented!("TResourceGroupView not yet implemented");
+        let mut group_read = match &self.latest_view {
+            ViewState::Sync(state) => {
+                state.read_from_group(group_key, resource_tag, self.txn_idx)?
+            },
+            ViewState::Unsync(state) => state
+                .unsync_map
+                .get_value_from_group(group_key, resource_tag),
```

Comment:
Shall we choose the same function names for syncmap and unsyncmap? Eg: read_from_group/get_value_from_group, read_group_size/get_group_size, ...